tests/test_gsf.py

```diff
@@ -882,7 +1188,7 @@ def test_loads_raises_when_head_too_small(self):
                                        b"\xd3\xe1\x91\xf0\x15\x94\x11\xe8\x91\xac\xdc\xa9\x04\x82N\xec" +
                                        b"\x00\x00\x00\x00\x00\x00\x00\x00")))
 
-        self.assertEqual(cm.exception.offset, 12)
+        self.assertEqual(cm.exception.offset, 51)
```

Comment:
Why has this changed here and below? Is the exception offset reporting the offset of the end of the block?

Reply:
I tweaked the exception reporting behaviour to highlight the start of the block that wouldn't parse rather than the start of the parent, I figured that's more useful in debugging. 
In this case that's the start of the "segm" block on line 1186 - the "head" block is only long enough to fit "se" so it reports that block doesn't fit.